backend/libexecution/dval.ml

```diff
@@ -469,6 +477,8 @@ let rec unsafe_dval_of_yojson_v0 (json : Yojson.Safe.t) : dval =
       DOption (OptJust (unsafe_dval_of_yojson_v0 dv))
   | `Assoc [("type", `String "errorrail"); ("value", dv)] ->
       DErrorRail (unsafe_dval_of_yojson_v0 dv)
+  | `Assoc [("type", `String "tuple"); ("value", `List vs)] ->
+      DTuple (List.map ~f:unsafe_dval_of_yojson_v0 vs)
```

Comment:
> Great, we caught a bug!
> 
> This needs to be explicitly handled in https://github.com/darklang/dark/blob/main/backend/libexecution/dval.ml#L382

@pbiggar I tried to handle this here to pass the roundtrip test. But it is still failing with the same error.